rust_dev_preview/cloudwatchlogs/README.md

```diff
@@ -0,0 +1,81 @@
+# AWS SDK for Rust code examples for CloudWatch Logs
+
+## Purpose
+
+These examples demonstrate how to perform several Amazon CloudWatch Logs (CloudWatch Logs) operations using the developer preview version of the AWS SDK for Rust.
+
+Use CloudWatch Logs to monitor, store, and access your log files from Amazon Elastic Compute Cloud instances, AWS CloudTrail, or other sources.
+
+## Code examples
+
+- [Get log eventss](src/bin/get-log-events.rs) (GetLogEvents)
+- [List log groups](src/bin/list-log-groups.rs) (DescribeLogGroups)
+- [List log streams](src/bin/list-log-streams.rs) (DescribeLogStreams)
+
+## ⚠ Important
+
+- We recommend that you grant this code least privilege, 
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
+- This code has not been tested in all AWS Regions.
+  Some AWS services are available only in specific
+  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+- Running this code might result in charges to your AWS account.
+
+## Running the code examples
+
+### Prerequisites
+
+You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+
+### get-log-events
+
+This example lists the events for a CloudWatch log stream in the Region.
```

Comment:
I *think* "CloudWatch log stream" is not allowed, and "CloudWatch Logs log stream" is to be avoided, so maybe just:
...lists the events for a log stream...?
--or--
...list the events for a log stream from CloudWatch Logs...?

Reply:
I changed the passage to the first suggested one.